crates/oxc_parser/src/diagnostics.rs

```diff
@@ -758,3 +758,11 @@ pub struct AParameterPropertyCannotBeDeclaredUsingARestParameter(
 pub struct AnEnumMemberCannotHaveANumericName(
     #[label("An enum member cannot have a numeric name")] pub Span,
 );
+
+#[derive(Debug, Error, Diagnostic)]
+#[error("TS18007: JSX expressions may not use the comma operator. Did you mean to write an array?")]
```

Comment:
@Boshen is there an index for these codes? I managed to find this by Googling the error text, but couldn't find a source-of-truth

Reply:
It's inside the TypeScript source code.